Marketplace/Support/manage-marketplace-subscriptions.md

```diff
@@ -0,0 +1,94 @@
+
+All subscriptions are shown at the Subscription Order level. By clicking the **+ Show Billable Items** button the subscription will expand to show all billable items that are associated with the subscription. These billable items can include monthly, hourly, usage, and one-time charges. You can collapse the billable items by clicking the **- Hide Billable Items** button.
+
+  ![Marketplace Manager2](../images/MM2.png)
+
+**Managing Subscriptions:** There are two actions that can be executed in the Marketplace Manager; **Update** and **Terminate**.
+
+###Update Request
+
+The **Update** button allows customers to update their Marketplace subscriptions.
+
+The Update option can be used for various use cases, below are a few examples.  
+
+If a customer wants to add additional quantities to a billable item in their subscription:
+Ex. Customer has a 10 license seat subscription and wants to add 10 additional 10 license seats.
+
+The customer can also use the Update option to downgrade their subscription:
+Ex. Customer has a 100 license seat subscription and wants to downgrade to a 80 license seat subscription.
+
+The downgrade option can also be used if a customer wants to terminate only some of the billable items for the subscription:
+Ex. Customer has a subscription that has a core product with multiple add-ons and customer wants to terminate one or all of the add-ons but NOT the core product.
+
+####Steps to Make an Update Request
+
+1. To update a subscription click on the **Update** button. The form below will populate.
+
+  ![Marketplace Manager3](../images/MM3.png)
+
+2. Next, fill out the form. The **Contact Email** will auto-populate, however, you can change it to another email if you chose.
+Fill out the **Request Description** field with your request. The request description should state which action you are looking to request.
+
+The options are:
+
+a. Update Billable Items
+b. Terminate Billable Items
+c. Add Billable Items *Additional charges may apply to certain selections*
+
+Ex. I am requesting an Update Billable Items to downgrade from 100 to 80 license seats.
+Ex. I am requesting to Terminate Billable Items.
+Ex. I am requesting to Add Billable Items to my subscription. Please add 10 license seats.
+
+3. Lastly, you will check the billable item(s) that you are wanting to update from the dropdown list that matches the update option that you are requesting.
+
+  ![Marketplace Manager4](../images/MM4.png)
+
+4. Click **Request Update** button to submit the request. You will receive a conformation email that your request has been received. *Please note: Submitting an Update Request is a manual process, it does not automatically alter functionality or billing of the product.*
+
+5. The Provider of the Service that you requested an update for will contact you via email to communicate that your request has been fulfilled and the date that the new functionality and updated billing will begin.
```

Comment:
The Provider of the Service for which you requested an update will contact you via email.  They will communicate to you when your request has been fulfilled, including when the new functionality and updated billing will begin.